File: solutions/leetcode/1480-running_sum_of_1d_array.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        sum = 0
        nnums = []

        for num in nums:
            sum += num
            nnums.append(sum)

        return nnums
        

File: solutions/leetcode/1480-running_sum_of_1d_array.java
class Solution {
    public int[] runningSum(int[] nums) {
        int sum = 0;
        int[] nnums = new int[nums.length];

        for(int i = 0; i < nums.length; i++){
            sum += nums[i];
            nnums[i] = sum;
        }
        return nnums;
    }
}